src/pages/home/Home.tsx

```diff
@@ -54,17 +55,19 @@ const Home: FunctionComponent<HomePropTypes> = (props) => {
       <Box
         data-testid="contentBox"
         justifyContent="center"
-        maxHeight={ height && height - 295 }
+        maxHeight={ height && height - 290 }
         overflow={ "scroll" }
         id="content"
       >
         <TabPanel value={ value } page={ Page.songs }>
-          <Box sx={ { marginLeft: "auto", marginRight: "auto", width: "1060px" } }>
+          <Container maxWidth="lg">
```

Comment:
I am curious to know what the difference is between Container and Box

Reply:
It looks like container is just designed to be the basic center aligned container element for layouts.

Reply:
I think the main difference is that you can use the theme assigned breakpoints ("sm", "md", etc...) instead of fixed pixel widths.